build.gradle

```diff
@@ -7,52 +7,76 @@
 // PLUGINS
 // -----------------------------------------------------------------------------
 
+buildscript {
+    repositories {
+        maven {
+            url 'https://plugins.gradle.org/m2/'
+        }
+    }
+    dependencies {
+        classpath 'io.github.gradle-nexus:publish-plugin:1.3.0'
+    }
+}
 plugins {
     // Apply the brightSPARK Labs plugin for standardisation
-    id 'com.brightsparklabs.gradle.baseline' version "0.1.0"
+    id 'com.brightsparklabs.gradle.baseline' version "4.0.0"
 
     // Apply the java-library plugin to add support for Java Library
     id 'java-library'
 
     // Apply the groovy plugin to also add support for Groovy (needed for Spock)
     id 'groovy'
 
-    // Apply bintray plugin for publishing artifacts
-    id "com.jfrog.bintray" version "1.8.4"
-    id "maven-publish"
+    // -----------------------------------------------------------------------------
+    // PUBLISHING
+    // -----------------------------------------------------------------------------
+
+    id 'maven-publish'
+    // Maven Central requires artifacts to be signed.
+    id 'signing'
+}
+
+/*
+ * Maven Central requires artifacts to be released via Sonatype Nexus.
+ * The Nexus Publish Plugin can only be applied to the root project.
+ * Do not apply it if this project is being included as a subproject.
+ */
+def isRootProject = project.path == ':'
+if (isRootProject) {
+    apply plugin: 'io.github.gradle-nexus.publish-plugin'
 }
 
-sourceCompatibility = 1.8
-targetCompatibility = 1.8
+sourceCompatibility = 17
+targetCompatibility = 17
 
 // -----------------------------------------------------------------------------
 // PROJECT IDENTIFICATION
 // -----------------------------------------------------------------------------
 
 project.description  = 'Dropwizard bundle for trusting externally authenticated users'
 project.ext.url      = 'https://github.com/brightsparklabs/dropwizard-external-auth-bundle'
-project.ext.scm      = 'https://github.com/brightsparklabs/dropwizard-external-auth-bundle.git'
-
+project.ext.scm      = 'scm:git@github.com:brightsparklabs/dropwizard-external-auth-bundle.git'
+project.group       = 'com.brightsparklabs'
+project.version     = 'git describe --always --dirty'.execute().text.trim()
```

Comment:
You shouldn't need to set `project.version` or `project.group`, the BSL baseline plugin will do it for you.